redhat-developer/app-services-cli
# fix(serviceaccount): update regex pattern for description

Follows up to https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/172

### Verification Steps

Create a service account with a description with ",", ".", "-" characters.
It should get created.

```shell
$ rhoas serviceaccount create --name "test" --overwrite --file-format env --description "Hello...- ,"

Service account "test" created successfully with ID "55f8d0ed-783d-4af9-b284-019791425da5".
Credentials saved to /home/ephelan/code/github.com/redhat-developer/app-services-cli/.env
```

### Type of change


### Checklist
